api/src/main/resources/messages.properties

```diff
@@ -1,3 +1,8 @@
 ${project.parent.artifactId}.title=Legacy UI Module
 ${project.parent.artifactId}.manage=Manage module
 ${project.parent.artifactId}.patient.duplicateName.voided={0} is a duplicate name for the same patient and it's voided. Please restore the existing name instead of creating new name
+
+legacyui.manageuser.providerAccount=Provider Account
```

Comment:
I spoke to Darius on IRC regarding prefixing module names and we mutually agreed to keep it this way. Hope it's alright now